data/ui/preferences.ui

```diff
@@ -51,13 +51,38 @@
           </packing>
         </child>
         <child>
-          <!-- n-columns=3 n-rows=5 -->
+          <!-- n-columns=2 n-rows=7 -->
           <object class="GtkGrid">
             <property name="visible">True</property>
             <property name="can-focus">False</property>
             <property name="row-spacing">2</property>
             <property name="row-homogeneous">True</property>
             <property name="column-homogeneous">True</property>
+            <child>
+              <object class="GtkLabel">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="tooltip-text" translatable="yes" context="program_language_tooltip">The preferred language on Minigalaxy start</property>
+                <property name="halign">start</property>
+                <property name="valign">center</property>
+                <property name="label" translatable="yes" context="program_language" comments="Program language. Ends with &quot;: &quot;.">Program language: </property>
```

Comment:
I think this field should probably be named "Language" and the field below it should be "Preferred game language".